### PR TITLE
[docker-orchagent]: Properly manage with supervisord

### PR DESCRIPTION
**Pull request by jleveque**
This allows supervisord to log individual process exit events to syslog.

### CODE REVIEW

File: dockers/docker-orchagent/Dockerfile.j2 [hunk 1]
```diff
@@ -26,7 +26,7 @@ debs/{{ deb }}{{' '}}
 RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
 RUN rm -rf /debs
 
-COPY ["start.sh", "orchagent.sh", "/usr/bin/"]
+COPY ["start.sh", "orchagent.sh", "swssconfig.sh", "/usr/bin/"]
```
Comment (stcheng) [inline dockers/docker-orchagent/Dockerfile.j2]: to rename it just to config.sh? swssconfig.sh will be confused with swssconfig script.

Comment (jleveque) [inline dockers/docker-orchagent/Dockerfile.j2]: @stcheng: I've been using this naming convention for supervisord. Basically, if a process requires more than one command to start properly, I've been wrapping them in a shell script named after the process. I think "config.sh" is not descriptive enough and can't be used as a convention. Maybe "swssconfig_start.sh"?

Comment (stcheng) [inline dockers/docker-orchagent/Dockerfile.j2]: okay. i see your point.

Comment (jleveque) [inline dockers/docker-orchagent/Dockerfile.j2]: I see yours as well. This has the potential to get confusing in the future. I'll consider submitting a PR that changes the names of all of these new process start scripts at once.

**Comment by lguohan** [comment]
:shipit:

**Pull request merged**